python/py-xdg/Portfile

```diff
@@ -1,37 +1,27 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           python 1.0
-
-name                py-xdg
-python.rootname     pyxdg
-version             0.28
-license             LGPL-2
-categories-append   textproc
-maintainers         {aronnax @lpsinger} openmaintainer
-supported_archs     noarch
-platforms           {darwin any}
-
-description         Python module implementing various freedesktop.org standards
-long_description    \
-    PyXDG is a python library to access freedesktop.org standards. Currently \
-    supported standards are: \
-    \
-    *   Base Directory Specification Version 0.6 \
-    *   Menu Specification Version 1.0 \
-    *   Desktop Entry Specification Version 1.0 \
-    *   Icon Theme Specification Version 0.8 \
-    *   Recent File Spec 0.2 \
-    *   Shared-MIME-Database Specification 0.13
-
-homepage            https://freedesktop.org/wiki/Software/pyxdg
-
-checksums           sha256  3267bb3074e934df202af2ee0868575484108581e6f3cb006af1da35395e88b4 \
-                    rmd160  83e3915cd88675d3a254917a5c00d74bf60fec25 \
-                    size    77776
-
-python.versions     27 37 38 39 310 311
-
-if {${name} ne ${subport}} {
-    depends_build-append    port:py${python.version}-setuptools
-}
+PortSystem              1.0
+PortGroup               python 1.0
+
+name                    py-xdg
+version                 6.0.0
+revision                0
+
+supported_archs         noarch
+platforms               {darwin any}
+license                 ISC
+maintainers             {@catap korins.ky:kirill} {@RhetTbull gmail.com:rturnbull} openmaintainer
+
+description             Variables defined by the XDG Base Directory Specification
+long_description        {*}${description}
+
+homepage                https://github.com/srstevenson/xdg-base-dirs
```

Comment:
Consider using the github port group.

Reply:
I think that using PyPi to get pythons artifacts is cleaner and I'm using it when this option is available.

But you're question points me that I must add `xdg-base-dirs` to keep synchronized. Thanks!